src/training.rst

```diff
@@ -0,0 +1,20 @@
+.. |date| date:: %b %Y
+
+Training Resources
+==================
+
+Below are resources for learning more about ODK. Email support@getodk.org if you'd like to add a link to this list.
+
+.. warning::
+
+ ODK has frequent improvements and so these resources quickly fall out of date. We recommend you start the resources that are most recent.
```

Comment:
I think you should add something that's like "The docs here are authoritative. You can use these resources to get a different perspective but check them against the docs"

"We recommend you start the resources that are most recent." probably missing a with